pocs/scheduler/dispatch.py

```diff
@@ -46,11 +46,7 @@ def get_observation(self, time=None, show_all=False, reread_fields_file=False):
         valid_obs = {obs: 1.0 for obs in self.observations}
         best_obs = []
 
-        common_properties = {
-            'end_of_night': self.observer.tonight(time=time, horizon=-18 * u.degree)[-1],
-            'moon': get_moon(time, self.observer.location),
```

Comment:
Can you remove the `get_moon` import from the top of this file as it is no longer used.